handsontable/test/e2e/helpers/dom/element.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('DOM helpers', () => {
```

Comment:
Just one last thing 🤞 IMO, it would be better to move the spec file to the helper module. Right [here](https://github.com/handsontable/handsontable/tree/develop/handsontable/src/helpers/dom/__tests__) as `element.spec.js`.

Reply:
You're right, I somehow missed that 🤦